crates/extensions/tedge_mqtt_ext/src/lib.rs

```diff
@@ -137,6 +145,43 @@ impl MqttMessageBox {
         }
     }
 
+    fn split(&mut self) -> (FromPeers<'_>, ToPeers<'_>) {
```

Comment:
This `split` method as well as the whole `struct MqttMessageBox` can be removed, as they are used only by the `MqttActor`.